### PR TITLE
DPL-754 Add new field priority_level to sample table

### PR DESCRIPTION
**Pull request by Sangeetha-Bheeman**
Closes #

Changes proposed in this pull request:

*
*
* ...

### CODE REVIEW

File: db/migrate/20230720131714_add_fields_to_long_read_qc_results.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# Adds new optional fields to long_read_qc_result
+class AddFieldsToLongReadQcResults < ActiveRecord::Migration[7.0]
+  def change
+    change_table :long_read_qc_result, bulk: true do |t|
+      t.string :priority_level, comment: 'Priority level eg Medium, High etc'
```
Comment (stevieing) [inline db/migrate/20230720131714_add_fields_to_long_read_qc_results.rb:7]: Again @emrojo I think we need to check that these are relevant.

**Review by stevieing — CHANGES_REQUESTED** [review]
Could you update the tests please? to ensure the fields are included in the json. https://github.com/sanger/unified_warehouse/blob/develop/spec/models/pac_bio_run_spec.rb

File: db/migrate/20230720131714_add_fields_to_long_read_qc_results.rb (hunk 1 above)
Comment (stevieing) [reply db/migrate/20230720131714_add_fields_to_long_read_qc_results.rb:7]: Update from meeting 27/07: date_required_by and reason_for_priority are no longer required and can be removed from this schema. priority_level is not a qc_result attribute but should be moved to sample.

Comment (Sangeetha-Bheeman) [reply db/migrate/20230720131714_add_fields_to_long_read_qc_results.rb]: Done. Added the migration to add priority_level to sample table.

File: db/migrate/20230802153621_add_priority_level_to_sample.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+# Adding priority_level eg Medium, High etc to recieve this information
+# in a sample message from TOL through traction receptions endpoint
```
Comment (harrietc52) [inline db/migrate/20230802153621_add_priority_level_to_sample.rb:4]:
```suggestion
# in a sample message from TOL through traction qc_receptions endpoint
```

File: db/migrate/20230802153621_add_priority_level_to_sample.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+# Adding priority_level eg Medium, High etc to recieve this information
```
Comment (harrietc52) [inline db/migrate/20230802153621_add_priority_level_to_sample.rb:3]:
```suggestion
# Adding priority_level e.g. Medium, High etc to receive this information
```

File: db/migrate/20230802153621_add_priority_level_to_sample.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+# in a sample message from TOL through traction receptions endpoint
+class AddPriorityLevelToSample < ActiveRecord::Migration[7.0]
+  def change
+    change_table :sample, bulk: true do |t|
+      t.string :priority_level, comment: 'Priority level eg Medium, High etc'
```
Comment (harrietc52) [inline db/migrate/20230802153621_add_priority_level_to_sample.rb:8]:
```suggestion
      t.string :priority_level, comment: 'Priority level e.g. Medium, High etc'
```

File: .github/workflows/automated_release.yml [hunk 5]
```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - master
       - develop
+      - dpl-754-add-fields-to-long-read-qc-results
```
Comment (emrojo) [inline .github/workflows/automated_release.yml:8]: This line won't be needed

Comment (emrojo) [inline .github/workflows/automated_release.yml:8]:
```suggestion
```

**Review by emrojo — APPROVED** [review]

**Pull request merged**